kafka/kafka.go

```diff
@@ -168,3 +189,73 @@ func (t *topicSync) PublishBatch(ctx context.Context, batch []*bps.PubMessage) e
 	}
 	return t.producer.SendMessages(pmsgs)
 }
+
+// --------------------------------------------------------------------
+
+// Subscriber wraps a kafka consumer and implements the bps.Subscriber interface.
+// It starts consuming from the newest offset (so from message, received after connecting to kafka).
+type Subscriber struct {
```

Comment:
does it need to be public? can't we just return the interface?

Reply:
Sure, we can - just followed existing approach with having structs for Sync/Async publisher.
Should I change them as well? (Publisher-s expose underlying producer, btw - don't think we should do that)

Reply:
I think it makes sense to have structs where there is something specific to add, i.e. where the `*kafka.Subscriber` has additional method. I don't mind keeping it as a struct, just didn't see the necessity

Reply:
I'll leave it as struct then - let's keep it uniform with Publisher (even if it doesn't expose anything additional (yet)).